src/DevBetterWeb.Core/Entities/BillingActivity.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System;
+using DevBetterWeb.Core.Interfaces;
 using DevBetterWeb.Core.SharedKernel;
 using DevBetterWeb.Core.ValueObjects;
 
 namespace DevBetterWeb.Core.Entities
 {
-  public class BillingActivity : BaseEntity
+  public class BillingActivity : BaseEntity, IAggregateRoot
```

Comment:
I wonder if BillingActivity makes sense as an aggregate root, or if it should simply be a collection on Member, which is already an Aggregate Root. But, I think we probably aren't using aggregates effectively yet in this solution so I'll probably let this go for this PR.